catalog/transformer/catalog-transformer-geojson-input/src/test/java/ddf/catalog/transformer/input/geojson/GeoJsonExtensibleTest.java

```diff
@@ -481,6 +483,47 @@ public void testBasicMetacardTypeNoMetacardType()
     verifyBasics(metacard);
   }
 
+  @Test
+  public void testTransformNoId() throws IOException, CatalogTransformerException {
+    Metacard metacard =
+        transformer.transform(new ByteArrayInputStream(sampleBasicMetacard().getBytes()));
+
+    assertEquals("ddf.metacard", metacard.getMetacardType().getName());
+    assertEquals("myId", metacard.getId());
+  }
+
+  @Test
+  public void testNonEmptyPropertyType() throws IOException, CatalogTransformerException {
+    Metacard metacard =
+        transformer.transform(new ByteArrayInputStream(sampleBasicMetacard().getBytes()), "myId");
+
+    assertEquals("ddf.metacard", metacard.getMetacardType().getName());
+  }
+
+  @Test
+  public void testEmptyPropertyTypeWithNonNullTransformers()
+      throws IOException, CatalogTransformerException {
+    transformer.setInputTransformers(new SortedServiceList());
+    Metacard metacard =
+        transformer.transform(
+            new ByteArrayInputStream(sampleBasicMetacardNoMetacardType().getBytes()), "myId");
+
+    assertEquals("ddf.metacard", metacard.getMetacardType().getName());
+    assertEquals("myTitle", metacard.getTitle());
+  }
+
+  @Test(expected = CatalogTransformerException.class)
+  public void testInvalidMetacardType() throws IOException, CatalogTransformerException {
+    MetacardType invalidMetacardType = new MetacardTypeImpl("invalid", new HashSet<>());
+    transformer.setMetacardTypes(Collections.singletonList(invalidMetacardType));
+    transformer.transform(new ByteArrayInputStream(sampleBasicMetacard().getBytes()), "myId");
+  }
+
+  @Test(expected = CatalogTransformerException.class)
```

Comment:
✏️ Consider verifying the message of the exception for this test and the one above.